### PR TITLE
Mentor complete profile

### PR DESCRIPTION
**Pull request by bfpepper**
Adds a new migration for profile_complete to Mentors table and the start of the logic to pull only mentor profile which are complete.

### CODE REVIEW

**Comment by neight-allen** [comment]
These PRs have gotten a lot better.

File: app/controllers/mentors_controller.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class MentorsController < ApplicationController
   def index
+    mentor = Mentor.where(profile_complete: true)
   end
```
Comment (meganft) [inline app/controllers/mentors_controller.rb]: @bfpepper We are actually not passing anything through this controller, it's all done with ajax, so this won't work. The rest of these changes make sense, but we have to do the filtering on the component and can't pass it in from here. 
@jbkimble

**Comment by vidoseaver** [comment]
@bfpepper whats going on w/ this?

File: app/controllers/api/v1/mentors_controller.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@ class Api::V1::MentorsController < Api::V1::BaseController
   skip_before_action :verify_authenticity_token
 
   def index
-    render json: Mentor.all
+    render json: Mentor.where(profile_complete: true)
```
Comment (meganft) [inline app/controllers/api/v1/mentors_controller.rb]: 👍

**Comment by meganft** [comment]
Looks good @bfpepper

**Pull request merged**